.dockerignore

```diff
@@ -2,6 +2,8 @@ _vendor
 storage/engine/engine.pc
 cockroach
 build/deploy
+.git/FETCH_HEAD
+.git/modules
```

Comment:
Now that you got rid of the `git submodule update`, do we need the `.git` directory at all?